zipkin/scribe_writer.py

```diff
@@ -13,14 +13,20 @@ def __init__(self, host, port,  category='zipkin'):
         self.client = Scribe.Client(iprot=protocol, oprot=protocol)
         self.category = category
 
-    def write(self, span):
+    def __span_to_entry(self, span):
         data = self.__build_log_message(span)
         message = "\\n".join([r for r in data.split("\n") if r != ""]) + "\n"
+        return Scribe.LogEntry(category=self.category, message=message)
+
+    def write_multiple(self, spans):
+        entries = [self.__span_to_entry(span) for span in spans]
         self.transport.open()
-        entry = Scribe.LogEntry(category=self.category, message=message)
-        self.client.Log([entry])
+        self.client.Log(entries)
```

Comment:
sweet, i thought you'd actually have to modify the client to accept multiple messages

LGTM